src/azure-cli-telemetry/azure/cli/telemetry/components/telemetry_client.py

```diff
@@ -39,6 +39,7 @@ def add(self, raw, flush=False, force=False):
             if not client:
                 continue
 
+            self._logger.info('Add %d record(s) to client %s for uploading: %s', len(records), instrumentation_key, records)
```

Comment:
Will `$HOMEDIR/.azure/logs/telemetry.log` be rotated? We certainly don't want it to grow indefinitely as time passes.